examples/corporate-profile/inspec.yml

```diff
@@ -0,0 +1,21 @@
+name: corporate-profile
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: All Rights Reserved
+summary: An InSpec Compliance Profile
+version: 0.1.0
+depends:
+  - name: ssh-hardening
+    supermarket: hardening/ssh-hardening
+  - name: os-hardening
+    url: https://github.com/dev-sec/tests-os-hardening/archive/master.zip
+  - name: ssl-benchmark
```

Comment:
ssl-benchmark is twice here, revealing an interesting bug, PR incoming for that.